build_extra/rust/run.py

```diff
@@ -28,6 +28,7 @@
 # When building with Cargo this variable is set by build.rs.
 env["GN_OUT_DIR"] = os.path.abspath(".")
 assert os.path.isdir(env["GN_OUT_DIR"])
+env["OUT_DIR"] = os.path.abspath(".")
```

Comment:
Add it to the appropriate GN target: `env += [ "OUT_DIR=." ]` or use --generated-source-dir.

Reply:
done